ui/component/scheduledStreams/view.jsx

```diff
@@ -21,8 +21,10 @@ type Props = {
   // --- perform ---
   setClientSetting: (string, boolean | string | number, boolean) => void,
   doShowSnackBar: (string) => void,
+  setLoaded: (boolean) => void,
```

Comment:
Yeah, the idea was to prevent the shift when scheduled livestreams get added to the homepage, but Tom has mentioned that these calls are about to change in the near future and that this isn't necessary. Will clean it up.